docs/content/TreeView.mdx

```diff
@@ -10,63 +10,98 @@ description: A hierarchical list of items where nested items can be expanded and
 ### File tree navigation without directory links
 
 ```jsx live drafts
-<nav aria-label="File navigation">
-  <TreeView aria-label="File navigation">
-    <TreeView.Item>
-      src
-      <TreeView.SubTree>
-        <TreeView.LinkItem href="#">Avatar.tsx</TreeView.LinkItem>
-        <TreeView.Item>
-          Button
-          <TreeView.SubTree>
-            <TreeView.LinkItem href="#" current>
-              Button.tsx
-            </TreeView.LinkItem>
-            <TreeView.LinkItem href="#">Button.test.tsx</TreeView.LinkItem>
-          </TreeView.SubTree>
-        </TreeView.Item>
-      </TreeView.SubTree>
-    </TreeView.Item>
-    <TreeView.Item>
-      public
-      <TreeView.SubTree>
-        <TreeView.LinkItem href="#">index.html</TreeView.LinkItem>
-        <TreeView.LinkItem href="#">favicon.ico</TreeView.LinkItem>
-      </TreeView.SubTree>
-    </TreeView.Item>
-    <TreeView.LinkItem href="#">package.json</TreeView.LinkItem>
-  </TreeView>
-</nav>
+<Box sx={{maxWidth: 320}}>
+  <nav aria-label="File navigation">
+    <TreeView aria-label="File navigation">
+      <TreeView.Item>
+        <TreeView.LeadingVisual>
+          <TreeView.DirectoryIcon />
+        </TreeView.LeadingVisual>
+        src
+        <TreeView.SubTree>
+          <TreeView.LinkItem href="#">
+            <TreeView.LeadingVisual>
+              <FileIcon />
+            </TreeView.LeadingVisual>
+            Avatar.tsx
+            <TreeView.TrailingVisual>
+              <StyledOcticon icon={DiffAddedIcon} color="success.fg" />
```

Comment:
Since these trailing visuals are used to visually communicate diff status, I'm pretty sure they should have an `aria-label`.

cc @ericwbailey

Reply:
That makes sense. How should the `aria-label` for the trailing visual impact the `aria-label` of the treeitem?

Reply:
Currently, the aria-label would be `Avatar.tsx added`. That seems correct but I'd love confirmation from @ericwbailey.

Reply:
I updated the docs and stories to include `aria-label` on these trailing visuals 👍

Reply:
Sorry to get to this after merging. I try to avoid using `aria-label` on non-interactive elements, given their mixed support. In the API guidance I am [recommending content that uses both `aria-hidden="true"` and a `sr-only` class applied to it](https://github.com/github/primer/pull/1321/files#diff-6d582e9797a0fa523d2bb3ed0ba3ed1739200d6d3b37b9e649fd348c494a9a2bR842-R847). 

In addition to the mixed support concern, we're also going to be using the text string to construct `aria-labeledby` announcements for the relevant TreeView node.